vnext/PropertySheets/React.Cpp.props

```diff
@@ -11,7 +11,7 @@
 
   <PropertyGroup Label="Configuration">
     <PlatformToolset Condition="'$(ReactNativeWindows_PlatformToolset)' != ''">$(ReactNativeWindows_PlatformToolset)</PlatformToolset>
-    <PlatformToolset Condition="'$(ReactNativeWindows_PlatformToolset)' == ''">v141</PlatformToolset>
+    <PlatformToolset Condition="'$(ReactNativeWindows_PlatformToolset)' == ''">v142</PlatformToolset>
     <GenerateProjectSpecificOutputFolder>false</GenerateProjectSpecificOutputFolder>
```

Comment:
shouldn't need this, remove

Reply:
This may be out of scope, but variable `ReactNativeWindows_PlatformToolset` has no reason to exist. `PlatformToolset` is predefined and overridable.

Reply:
I'll try to remove it.